examples/e2e-tests/kubernetes/release/default/definition.json

```diff
@@ -9,31 +9,214 @@
           {
             "name": "tiller",
             "enabled": true,
-            "config": {
-              "max-history": "10"
-            },
             "containers": [
               {
                 "name": "tiller",
+                "image": "gcr.io/kubernetes-helm/tiller:v2.8.1",
```

Comment:
Curious. Why are we doing this? Defaults should populate the exact same thing

Reply:
I'm testing